app/dashboard/organization/history/acceptedGrant/[id]/page.tsx

```diff
@@ -0,0 +1,148 @@
+"use client";
+import PageHeading from "@/components/dashboard/PageHeading";
+import { useState, useEffect } from "react";
+import ContentCopyIcon from "@mui/icons-material/ContentCopy";
+import CheckIcon from "@mui/icons-material/Check";
+import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
+import Link from "next/link";
+import { getAcceptedGrantData, getAcceptedApplicationByGrantID } from "@/utils/scripts/applications";
+import { useToast } from "@/components/ui/use-toast";
+import { set } from "date-fns";
```

Comment:
Unused, is this still needed?

Reply:
im dead idek what that function does, not sure how that got there lmao